amy/emails/urls.py

```diff
@@ -5,13 +5,38 @@
 urlpatterns = [
     path(
         "templates/",
-        views.EmailTemplateListView.as_view(),
-        name="email_templates_list",
+        include([
+            path(
+                "",
+                views.EmailTemplateListView.as_view(),
+                name="email_templates_list",
+            ),
+            path(
+                "create/",
+                views.EmailTemplateCreateView.as_view(),
+                name="email_template_create",
+            ),
+        ]),
     ),
     path(
         "template/<uuid:pk>/",
-        views.EmailTemplateDetailView.as_view(),
-        name="email_template_detail",
+        include([
+            path(
+                "",
+                views.EmailTemplateDetailView.as_view(),
```

Comment:
Another convention point - we don't usually include `View` in our view class names, and detail views are typically `ModelNameDetails` (plural)